src/main/java/org/zephyrsoft/sdb2/remote/Patch.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of the Song Database (SDB).
+ *
+ * SDB is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * SDB is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with SDB. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.zephyrsoft.sdb2.remote;
+
+import org.zephyrsoft.sdb2.model.Persistable;
+import org.zephyrsoft.sdb2.model.SongsModel;
+
+import jakarta.xml.bind.annotation.XmlAccessOrder;
+import jakarta.xml.bind.annotation.XmlAccessType;
+import jakarta.xml.bind.annotation.XmlAccessorOrder;
+import jakarta.xml.bind.annotation.XmlAccessorType;
+import jakarta.xml.bind.annotation.XmlElement;
+import jakarta.xml.bind.annotation.XmlRootElement;
+
+/**
+ *
```

Comment:
Please don't create empty javadoc comments. If there's nothing to be said about the class, remove the comment completely.

Reply:
oh ok.